packages/mattermost-redux/src/utils/post_list.ts

```diff
@@ -343,10 +342,6 @@ export const postTypePriority = {
     [Posts.POST_TYPES.EPHEMERAL]: 16,
 };
 
-export function comparePostTypes(a: typeof postTypePriority, b: typeof postTypePriority) {
-    return postTypePriority[a.postType] - postTypePriority[b.postType];
```

Comment:
Could you also remove `postTypePriority` itself? It was only used by `comparePostTypes` and its tests